inlineplz/interfaces/__init__.py

```diff
@@ -5,6 +5,16 @@
 
 from inlineplz.interfaces.github import GitHubInterface
 
+STASH_SUPPORTED = False
+try:
+    from inlineplz.interfaces.stash import StashInterface
```

Comment:
hmm, could we instead do the try/except within the stash interface, wrapping stashy? and should we maybe just dump stashy and write a custom wrapper for the stash API endpoints we need?